worlds/tunic/__init__.py

```diff
@@ -502,6 +502,13 @@ def remove(self, state: CollectionState, item: Item) -> bool:
             state.tunic_need_to_reset_combat_from_remove[self.player] = True
         return change
 
+    def write_spoiler_header(self, spoiler_handle: TextIO):
+        if self.options.hexagon_quest and self.options.ability_shuffling:
+            spoiler_handle.write('Ability Unlocks (Hexagon Quest):\n')
```

Comment:
```suggestion
            spoiler_handle.write('\nAbility Unlocks (Hexagon Quest):\n')
```
Cause it looks a little weird to me being directly under the last option.
Also maybe use " instead of ' since the rest of the code does o: